exercises/concept/ellens-alien-game/alien_class_test.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+
+from alien_class import (
+    Alien,
+    new_alien_list
+)
+
```

Comment:
Yeah, I noticed that the concept test files had that decorator, while the practice files didn't.  They also seemed to be laid out kind of differently, which is more or less why I mentioned my test file's layout.  I think I was using a practice test layout rather than a concept test layout.